docs/content/en/docs/getting-started/install/_index.md

```diff
@@ -106,59 +108,7 @@ eksctl anywhere version
-connected to the internet.
-
-An airgapped environment is especially important if you require the most secure networks.
-EKS Anywhere supports airgapped installation for creating clusters using a registry mirror.
-For airgapped installation to work, the Admin machine must have:
-
-* Temporary access to the internet to download images and artifacts
-* Ample space (80 GB or more) to store artifacts locally
-
-
-To create a cluster in an airgapped environment, perform the following:
-
-1. Download the artifacts and images that will be used by the cluster nodes to the Admin machine using the following command:
-   ```bash
-   eksctl anywhere download artifacts
-   ```
-   A compressed file `eks-anywhere-downloads.tar.gz` will be downloaded.
-
-1. To decompress this file, use the following command:
-   ```bash
-   tar -xvf eks-anywhere-downloads.tar.gz
-   ```
-   This will create an eks-anywhere-downloads folder that we’ll be using later.
-
-1. In order for the next command to run smoothly, ensure that Docker has been pre-installed and is running. Then run the following:
-   ```bash
-   eksctl anywhere download images -o images.tar
-   ```
-
-   **For the remaining steps, the Admin machine no longer needs to be connected to the internet or the bastion host.**
-
-1. Next, you will need to set up a local registry mirror to host the downloaded EKS Anywhere images. In order to set one up, refer to [Registry Mirror configuration.]({{< relref "../optional/registrymirror.md" >}})
-
-1. Now that you’ve configured your local registry mirror, you will need to import images to the local registry mirror using the following command (be sure to replace <registryUrl> with the url of the local registry mirror you created in step 4):
-   ```bash
-   eksctl anywhere import images -i images.tar -r <registryUrl> \
-      -- bundles ./eks-anywhere-downloads/bundle-release.yaml
-   ```
-You are now ready to deploy a cluster by following instructions on the provider you select from the [providers]({{< relref "/docs/getting-started/chooseprovider/" >}}). See text below for specific provider instructions.
-
-### For Bare Metal (Tinkerbell)
-You will need to have hookOS and its OS artifacts downloaded and served locally from an HTTP file server.
-You will also need to modify the [hookImagesURLPath]({{< relref "../baremetal/bare-spec/#hookimagesurlpath" >}}) and the [osImageURL]({{< relref "../baremetal/bare-spec/#osimageurl" >}}) in the cluster configuration files.
-Ensure that structure of the files is set up as described in [hookImagesURLPath.]({{< relref "../baremetal/bare-spec/#hookimagesurlpath" >}})
-
-### For vSphere
-If you are using the vSphere provider, be sure that the requirements in the
-[Prerequisite checklist]({{< relref "../vsphere/vsphere-prereq" >}}) have been met.
+For more information on how to prepare the Administrative machine for airgapped environments, go to the [Airgapped](/docs/getting-started/airgapped/) page. 
```

Comment:
Thanks for removing this sections. That should have been done when we did the reorg a while back.